litewallet/Environment.swift

```diff
@@ -143,3 +144,23 @@ struct E {
 		return UIScreen.main.bounds.size.height
 	}
 }
+
+/// Remote Config
+enum EnvironmentRemoteConfigKey: String, Equatable {
+	case k20241212_API_1
+	case k20241212_API_2
+
+	var key: String {
+		switch self {
+		case .k20241212_API_1: return "key_api_baseurl_dev_new_enabled"
+		case .k20241212_API_2: return "key_api_baseurl_prod_new_enabled"
+		}
+	}
+
+//	var type: Any {
```

Comment:
Hi @kcw-grunt is there any reason to keep this commented code?

Reply:
Good catch @azisramdhan .  Refactoring